lib/vsc/install/shared_setup.py

```diff
@@ -1431,28 +1428,25 @@ def parse_target(self, target, urltemplate=None):
             ]
         else:
             urls = [('github.com', 'git+https://')]
+
         for dependency in set(new_target['install_requires'] + new_target['setup_requires'] +
             new_target['tests_require']):
+
+            # see https://docs.python.org/3/reference/expressions.html#comparisons
+            split_re = r'(\s)?([<>]=?|[=!]=)(\s)?'
```

Comment:
@wpoely86 Since you're using this same regex in multiple places, why not compile it here and then use it below?

```python
split_re = re.compile(r'(\s)?([<>]=?|[=!]=)(\s)?')